.travis.yml

```diff
@@ -110,6 +96,17 @@ install:
         make --no-print-directory -s pkg-utils pkg-kmod || travis_terminate 1;
         sudo dpkg -i *.deb || travis_terminate 1;
       fi
+    # Return to libcstor code to complie zrepl which is responsible for uZFS feature.
+    # Save the current location to get back
+    - pushd .
+    - cd ..
```

Comment:
we can just do with one 'cd' command, and after doing 'make', popd can be performed. No need to do 'cd ../..'

Reply:
Updated PTAL